src/components/alert/__tests__/Alert.Test.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+
+import Alert from '../index';
+
+const MockIcon = () => <svg data-testid="test-icon" />;
+
+describe('<Alert>', () => {
+  it('renders Alert with title', () => {
+    render(
+      <Alert
+        title="I am the title"
+        typeMessage="info"
+        description="I am a description"
+        icon={<MockIcon />}
+      />
+    );
+
+    expect(screen.getByText('I am the title')).toBeInTheDocument();
+  });
+
+  it('renders Alert with link', () => {
```

Comment:
I think the other tests besides this one are not really testing something so I would drop those. I might be mistaken, let's discuss in slack if you don't agree

Reply:
I have adjusted the tests. Thank you!